### ISSUE TITLE
Show "no results" when there is no results

### ISSUE THREAD

**Issue by kevinsimper** [redex/redex.github.io#18]
Right now when you search and there is no results, nothing happends. It would be great if you would get a messaging saying, "No results".

**Comment by glennsl** [comment]
Yeah, this has been on my todo list for a long time. Not sure why I didn't just implement it from the start, since it seem pretty easy to add. But I'd happily accept a PR if you want to have a go at it.
